src/Contract/Config/Analyser.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Contract\Config;
+
+final class Analyser
```

Comment:
AnalyserConfig, would be the NamingConvention of he other Configs.

Reply:
Ok done. I was following the example of the Layer class.

Reply:
@gennadigennadigennadi Hey, would you contact me by email? You can reach me at dkinzler(_AT_)wikimedia.org. I would like to mak deptrac part of the CI pipeline for MediaWiki.